.github/workflows/npm-release-test.yml

```
@@ -0,0 +1,143 @@
+on:
+  release:
+    types: [published]
+
+  workflow_dispatch:
+
+  pull_request:
+    types: [labeled, synchronize]
+
+permissions:
+  contents: read
+  id-token: read
```

actionlint (1.7.12)

[error] "read" is invalid as permission of scope "id-token". available values are "write", "none"

(permissions)